src/colorizer/loaders/UrlArrayLoader.ts

```diff
@@ -86,13 +86,15 @@ export default class UrlArrayLoader implements IArrayLoader {
       await parquetRead({
         file: arrayBuffer,
         compressors,
+        columns: ["data"],
         onComplete: (loadedData: number[][]) => {
-          for (const row of loadedData) {
-            dataMin = dataMin === undefined ? row[0] : Math.min(dataMin, row[0]);
-            dataMax = dataMax === undefined ? row[0] : Math.max(dataMax, row[0]);
-            data.push(row[0]);
+          data = Array(loadedData.length);
+          for (let i = 0; i < loadedData.length; i++) {
+            const value = Number(loadedData[i][0]);
```

Comment:
Is there any part of the code that ensures `loadedData[i]` is defined and has a length greater than 0?

Reply:
That's a good question! Not currently, though this is parsing output from the `hyparquet` library which I think should be stable. Just to be safe I'll add a fallback for `loadedData[i][0]`.